oemoflex/facades.py

```diff
@@ -532,7 +538,7 @@ def build_solph_components(self):
         self.subnodes = (inflow, internal_bus, pump)
 
 
-class ExtractionTurbine(ExtractionTurbineCHP, Facade):
+class ExtractionTurbine(ExtractionTurbineCHP, Facade):  # pylint: disable=too-many-ancestors
```

Comment:
Yeah, no idea how we could else deal with this. oemof.tabular does not use pylint, I guess, or ignores it as well?